web/ocr-fileformat.js

```diff
@@ -45,7 +45,6 @@ function escapeHTML(str) {
         replace(/</g, '&lt;').
         replace(/"/g, '&quot;').
         replace(/'/g, '&#39;').
-        replace(/\//g, '&#x2F').
```

Comment:
Maybe adding a semi-colon is enough here...

Reply:
Great, yes. https://digi.bib.uni-mannheim.de/ocr-fileformat-test/ now includes that fix.